src/components/app/routes/loaders/rootLoader.ts

```diff
@@ -73,7 +73,7 @@ export default function makeRootLoader(queryClient) {
 
     // Redirect user to search page, for first-time users with no assignments.
     redirectToSearchPageForNewUser({
-      enterpriseSlug,
+      enterpriseSlug: enterpriseSlug as string,
```

Comment:
[nit] possible test data left in?

Reply:
Now that this is TypeScript, it knows `enterpriseSlug` is either undefined _or_ a string based on the `params` object type. The `enterpriseSlug as string` is coercing the value as a string given the `redirectToSearchPageForNewUser` does not accept `enterpriseSlug` as undefined, thus throwing a TypeScript error. In this part of the code, `enterpriseSlug` is known to only be a string, so this is telling TypeScript that.

Reply:
Ohh yeah I am just noticing its not wrapped in a set of quotation marks denoting as a string 😅 .